File: c++/2154.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
  freopen("example.txt", "r", stdin);
  int N;
  cin >> N;

  string s = "";

  for (int i = 1; i <= N; i++) {
    s += to_string(i);
  }

  int result = s.find(to_string(N));

  cout << result + 1;

  return 0;
}

File: js/2154.js
const fs = require('fs');

const N = +fs.readFileSync('example.txt').toString().trim();

let s = '';

for (let i = 1; i <= N; i++) {
  s += i.toString();
}

const result = s.indexOf(N.toString());

console.log(result + 1);
